### PR TITLE
Fix bug when selecting frequent licenses

### PR DESCRIPTION
**Pull request by nicarl**
There was some hidden magic that converted the license name the user wrote to uppercase letters if there was a match with a frequent license. This logic is removed and the user has to actively choose the frequent license now.
Only in the case of a perfect match (user input: MIT, frequent license: MIT) the frequent license is automatically chosen.
Also removed some formatting, which was confusing for the user and added an integration test.

### CODE REVIEW

File: src/Frontend/integration-tests/__tests__/navigate-around.test.tsx [hunk 1]
```diff
@@ -229,4 +229,48 @@ describe('The App integration', () => {
...
+    screen.getByText('gpl-2.0');
+    expect(screen.queryByText('frequent license')).toBeFalsy();
+
+    goToView(screen, View.Audit);
+    insertValueIntoTextBox(screen, 'License Name', 'GPL-2.0');
+
+    clickOnButton(screen, ButtonTitle.Save);
+    goToView(screen, View.Report);
+    screen.getByText('GPL-2.0');
+    screen.getByText('frequent license');
```
Comment (nicarl) [inline src/Frontend/integration-tests/__tests__/navigate-around.test.tsx:274]: Maybe also test the select feature of the dropdown?

**Review by benedikt-richter — APPROVED** [review]
👍

File: src/Frontend/integration-tests/__tests__/navigate-around.test.tsx (hunk 1 above)
Comment (nicarl) [reply src/Frontend/integration-tests/__tests__/navigate-around.test.tsx:275]: see part below

**Review by JakobSchubert — APPROVED** [review]

File: src/Frontend/Components/InputElements/AutoComplete.tsx [hunk 2]
```diff
@@ -46,8 +46,8 @@ export function AutoComplete(props: AutoCompleteProps): ReactElement {
 
   const inputValue = props.text || '';
   const inputValueIndexInOptions: number = props.options
-    .map((name) => name.toLowerCase())
-    .indexOf(inputValue.toLowerCase());
+    .map((name) => name)
```
[the remark this answers is not in the document]
Comment (nicarl) [reply src/Frontend/Components/InputElements/AutoComplete.tsx]: Good catch

**Pull request merged**